src/assets/stylesheets/Button.scss

```diff
@@ -98,31 +99,40 @@
 
   // Variants
   &--primary {
-    $colours: $rpf-teal-800, $rpf-teal-600, $rpf-teal-200, $rpf-grey-600,
-      $rpf-teal-800, $rpf-teal-900;
+    $colours: var(--rpf-button-primary-background-color),
+      var(--rpf-button-primary-background-color-active),
+      var(--rpf-button-primary-background-color-disabled),
+      var(--rpf-button-primary-color-disabled),
+      var(--rpf-button-primary-background-color-focus),
+      var(--rpf-button-primary-background-color-hover);
     @include button-styling($colours...);
   }
 
   &--secondary {
-    $colours: inherit, inherit, $rpf-grey-50, $rpf-grey-600, inherit, inherit;
+    $colours: inherit, inherit,
+      var(--rpf-button-secondary-background-color-disabled),
+      var(--rpf-button-secondary-background-color-active), inherit, inherit;
     @include button-styling($colours...);
 
-    border: 2px solid $rpf-teal-800;
+    border: 2px solid var(--rpf-button-secondary-background-color);
 
     &:active {
-      border: 2px solid $rpf-teal-900;
+      border: 2px solid var(--rpf-button-secondary-background-color-active);
     }
 
     &:hover {
-      color: $rpf-grey-600;
+      color: var(--rpf-button-secondary-background-color-hover);
+      border-color: var(
+        --rpf-button-secondary-background-color-hover
+      ) !important;
```

Comment:
What is this `!important` overriding? I'm assuming there's no way to avoid this? 🤔

Reply:
It overrides a hover colour, preventing a teal border from appearing within the project-site integrated editor

Reply:
One for a tidy-up ticket